specification.md

```diff
@@ -73,6 +76,29 @@ Represents the root object and is usually referred to as a WCIF.
 ```
 
 
+### Series
+
+Represents a Competition Series as per the WCA Competition Requirements Policy.
+
+| Attribute | Type | Description |
+| --- | --- | --- |
+| `id` | `String` | The unique Competition Series identifer. |
+| `name` | `String` | The full name of the Competition Series. |
+| `shortName` | `String` | A briefer version of `name`, may be the same if `name` is already short. |
+| `competitions` | `[String]` | The identifiers of all competitions that are part of this series, ordered by start date. **Includes the current competition represented by this WCIF**. |
+
+#### Example
+
+```json
+{
+  "id": "AwesomeAustralianSeries2022",
```

Comment:
Couldn't it just be the database id?

I don't think we need a human readable id in the WCIF, but if we really do I'd rather include it as a "slug" or "friendlyId".
This will avoid some annoying changes when doing (de)serialization, and will avoid us referring to "ids" without actually being sure if it's the db id or the human readable id.

Reply:
I don't think database IDs should be a part of an external API Interchange format. Including it as "slug" or "friendlyId" would also be inconsistent with other existing WCIF fields, like `Event` or even `Competition` itself.

Another point to consider: Once we have proper website support and more dedicated features for Series Competitions (right now, my PR over on the website repo is only a bare minimum implementation to save Delegates from the pain of using a spreadsheet, but eh, one can dream! :smile: ) they might become much more popular and wide-spread and people might want to refer to them by ID.

Also, in some other places where there are numeric IDs in the current WCIF standard (for example, everything under `Schedule`) they are actually auto-increment _within the current WCIF_. This might lead to confusion when we're using the ID field here as an auto-increment within our entire database. In contrast, the very few places where we _do_ expose auto-increments (registration IDs, user IDs) we prefix them as `wca{User|Registration}Id`.

Reply:
Kindly ping @viroulep :)

Reply:
Sorry for the delay Gregor!

The truth is that I spent a bit of time trying to figure out how we could have a "good" approach for including series in the WCIF, and I'm failing to come up with better than your proposal.
It's the first time we introduce "something" (ie: the other competitions of the series through their id) that is not fully included in the WCIF, which kinds of assume that the WCIF consumer needs to now about these competitions too.

But again I don't have a great suggestion on how to "fix" this; including the whole WCIF for these competitions definitely doesn't sound like the appropriate way to go.
On that note what do you think about changing `competitions` to `competitionIds` or `competitionsIds` (based on the fact it's actually an array of ids rather than an array of competitions)?

At this point since it's merged in the website already I wouldn't mind if this was merged too (but I'm still curious about your opinion about `competitions` vs `competitionIds` :p).

Reply:
`competitionIds` makes sense, especially to avoid implying that the key holds the entire WCIFs for all sibling competitions.